### DISCUSSION TITLE
Link in a table block

### DISCUSSION THREAD

**Discussion by LTCTHITA** [Budibase/budibase#11837] [Help] [answered]
Hey ...

How can I put a link in a table block?
I went to the budibase demo platform and saw a link in the table, but I don't know how to do it. Could you please tell me?
![image](https://github.com/Budibase/budibase/assets/93240913/20e6a153-fa0a-40f1-9bde-e838de0f1c4f)
![image](https://github.com/Budibase/budibase/assets/93240913/b8fd194c-b70a-4344-9bd8-340d9110814a)
 
Thank you!

**Comment by ConorWebb96** [comment]
Hey @LTCTHITA,

You will have to eject your block component and then you should be able to drag the link component inside the table after it has been ejected.

I hope this helps.

**Reply by LTCTHITA** [reply]
Hey Conor,
I don't understand. Could you please explain more?

Thank you,

**Reply by ConorWebb96** [reply, marked as answer]
Hey @LTCTHITA,

Can you click on your component called `tag block` and see whether or not you have the option to `eject block`. 

If it is already just a `Table` component then you can drag the link you have inside it by clicking the `new link` and dragging it inside the `tag block` component. Optionally you can use the shortcut keys; `command option up arrow` (Not sure what this would be on a windows machine) to move what ever component you've highlighted up. It should place this into the table component only if its not a table block component.

I hope this helps.
